.travis/check-posix-pty

```diff
@@ -129,6 +130,10 @@ EOF
     extaddr=$(awk '/extaddr/{getline; print}' $OT_OUTPUT | tr -d '\r\n')
     echo "Extended address is: ${extaddr}"
 
+    # Test join multicast group
+    NETIF_INDEX=$(ip link show "${VALID_NETIF_NAME}" | cut -f 1 -d ":" | head -n 1)
+    python3 "$(pwd)/.travis/test_multicast_join.py" "${NETIF_INDEX}" || die 'multicast group join failed'
```

Comment:
It seems this doesn't verify whether the multicast address is added to OpenThread's external multicast address list.